tests/framework/extensions/provisioninginput/config.go

```diff
@@ -171,4 +171,5 @@ type Config struct {
 	Registries             *Registries                              `json:"registries" yaml:"registries"`
 	UpgradeStrategy        *rkev1.ClusterUpgradeStrategy            `json:"upgradeStrategy" yaml:"upgradeStrategy"`
 	Advanced               *Advanced                                `json:"advanced" yaml:"advanced"`
+	SSHTests               []string                                 `json:"sshTests" yaml:"sshTests"`
```

Comment:
I'm not convinced this is getting picked up to run the test with the config I'm seeing used in jenkins.